Loop/Managers/DeviceAlert/InAppModalDeviceAlertPresenter.swift

```diff
@@ -156,4 +170,48 @@ extension InAppModalDeviceAlertPresenter {
         return controller
     }
     
+    private func playSound(for alert: DeviceAlert) {
+        guard let soundName = alert.soundName else { return }
+        switch soundName {
+        case .vibrate:
+            soundPlayer.vibrate()
+        case .silence:
+            break
+        default:
+            // Assuming in-app alerts should also vibrate.  That way, if the user has "silent mode" on, they still get
+            // some kind of haptic feedback
+            soundPlayer.vibrate()
+            guard let url = DeviceAlertManager.soundURL(for: alert) else { return }
+            soundPlayer.play(url: url)
+        }
+    }
+}
+
+private class AVSoundPlayer: AlertSoundPlayer {
+    private var soundEffect: AVAudioPlayer?
+    private let log = OSLog(category: "AVSoundPlayer")
+    
+    enum Error: Swift.Error {
+        case playFailed
+    }
```

Comment:
Is this type in use?

Reply:
Ah, don't need it anymore.  Removing, good eye! 👀